packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -59,7 +59,6 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
     name,
     onChange,
     onChangeCommitted,
-    onMouseDown,
```

Comment:
It doesn't have to be specified explicitly anymore, as all extra event handlers from props are passed into the `getRootProps`